sql/moz-fx-data-shared-prod/telemetry/active_users/view.sql

```diff
@@ -0,0 +1,189 @@
+  client_id,
+  CASE
+    WHEN isp_name = 'BrowserStack'
+      THEN CONCAT(app_name, ' ', isp_name)
+    WHEN distribution_id = 'MozillaOnline'
+      THEN CONCAT(app_name, ' ', distribution_id)
+    ELSE app_name
+  END AS app_name,
+  days_seen_bits,
+  days_active_bits,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) = 0 AS BOOLEAN) AS is_dau,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) <= 7 AS BOOLEAN) AS is_wau,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) AS BOOLEAN) AS is_mau,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) = 0 AS BOOLEAN) AS is_daily_user,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) <= 7 AS BOOLEAN) AS is_weekly_user,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) AS BOOLEAN) AS is_monthly_user,
+  CAST(1 AS BOOLEAN) AS is_desktop,
+  CAST(0 AS BOOLEAN) AS is_mobile
+FROM
+  `moz-fx-data-shared-prod.telemetry_derived.clients_last_seen_v2`
+UNION ALL
+  -- Fenix
+SELECT
+  submission_date,
+  client_id,
+  CASE
+    WHEN isp = 'BrowserStack'
+      THEN CONCAT('Fenix ', isp)
+    WHEN distribution_id = 'MozillaOnline'
+      THEN CONCAT('Fenix', ' ', distribution_id)
+    ELSE 'Fenix'
+  END AS app_name,
+  days_seen_bits,
+  days_active_bits,
+  CAST(NULL AS INT64) AS days_active_bits,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) = 0 AS BOOLEAN) AS is_dau,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) <= 7 AS BOOLEAN) AS is_wau,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) AS BOOLEAN) AS is_mau,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) = 0 AS BOOLEAN) AS is_daily_user,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) <= 7 AS BOOLEAN) AS is_weekly_user,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) AS BOOLEAN) AS is_monthly_user,
+  CAST(0 AS BOOLEAN) AS is_desktop,
+  CAST(1 AS BOOLEAN) AS is_mobile
+FROM
+  `moz-fx-data-shared-prod.fenix.baseline_clients_last_seen`
+LEFT JOIN
+  fenix_distribution_id
+  USING (client_id, sample_id, submission_date)
+UNION ALL
+-- Firefox iOS
+SELECT
+  submission_date,
+  client_id,
+  CASE
+    WHEN isp = 'BrowserStack'
+      THEN CONCAT('Firefox iOS', ' ', isp)
+    ELSE 'Firefox iOS'
+  END AS app_name,
+  days_seen_bits,
+  days_active_bits,
+  CAST(NULL AS INT64) AS days_active_bits,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) = 0 AS BOOLEAN) AS is_dau,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) <= 7 AS BOOLEAN) AS is_wau,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) AS BOOLEAN) AS is_mau,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) = 0 AS BOOLEAN) AS is_daily_user,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) <= 7 AS BOOLEAN) AS is_weekly_user,
+  CAST(mozfun.bits28.days_since_seen(days_seen_bits) AS BOOLEAN) AS is_monthly_user,
+  CAST(0 AS BOOLEAN) AS is_desktop,
+  CAST(1 AS BOOLEAN) AS is_mobile
+FROM
+  `moz-fx-data-shared-prod.firefox_ios.baseline_clients_last_seen`
+UNION ALL
+-- Klar Android
```

Comment:
Is Klar Android also required or shall we remove it from this view? cc/@bochocki

Reply:
I think we should include it but set the `is_mobile` column to `FALSE`.